website/src/content/docs/docs/libraries/xml/guide.md

```diff
@@ -278,6 +283,7 @@ Pet:
 ```tsp
 @encodedName("application/xml", "XmlPet")
 model Pet {
+  @encodedName("application/xml", "ItemsTags")
```

Comment:
I think maybe you don't want this change - this makes scenario 2.2 exactly the same as scenario 2.4